Gemfile

```diff
@@ -34,4 +33,9 @@ gem 'unicorn'
 group :development, :test do
   gem 'rspec-rails', '~> 3.0'
   gem 'rubocop', require: false
+  gem 'rubocop-rspec', require: false
+end
+
+group :test do
+  gem 'codeclimate-test-reporter', require: nil
```

Comment:
It's CodeClimate setup for usage on Travis CI.

See: https://docs.travis-ci.com/user/code-climate/